style.css

```diff
@@ -366,3 +533,114 @@ a { text-decoration: none; }
   list-style: none;
   padding: 0;
 }
+
+@media screen and (min-width: 620px) {
+  header {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 1200px) {
```

Comment:
- [ ] As mentioned by the previous reviewer, your desktop design also doesn't match the design template. Note that you are expected to stick to the figma design as much as possible, this includes padding ,margins, background images, colors and font size and style.
### your desktop views
![4](https://user-images.githubusercontent.com/94551484/213886607-5678bd39-6e8f-4f17-aa56-408e2866dd74.png)
![5](https://user-images.githubusercontent.com/94551484/213886609-f76c4bcc-e0fd-423c-a676-abfba16bfef5.png)
![6](https://user-images.githubusercontent.com/94551484/213886611-6167518a-d096-4450-803c-d60e99f13c8f.png)
![7](https://user-images.githubusercontent.com/94551484/213886613-f4c509e0-712c-4a2a-bb13-b1d65ea2eb1a.png)
### figma designs
![f4](https://user-images.githubusercontent.com/94551484/213886615-dc4c718c-4a97-40f6-926e-ec8bd2cfc07c.png)
![f5](https://user-images.githubusercontent.com/94551484/213886617-4e07d89a-c6d3-46d3-a379-118a87d99405.png)
![f6](https://user-images.githubusercontent.com/94551484/213886618-4ec8e277-a399-4445-9e82-14fbfa760cc2.png)
![f7](https://user-images.githubusercontent.com/94551484/213886620-1252dc9b-578a-452f-ab2f-0564a87a379c.png)